index.css

```diff
@@ -212,4 +214,54 @@ footer {
 
 footer .license {
 	float: right;
-}
\ No newline at end of file
+}
+
+footer br{
+	display: none;
+}
+
+h2{
+	text-align: center;
+}
+
+@media only screen and (max-width: 77em){
+	main{	
+		width: 100%;
+		max-width: 52em;
```

Comment:
This is just a cryptic way of writing `width: 52em;`.

Reply:
The only reason for using max-width instead of width is to make the sizes responsive.

Reply:
Main will be 52 em wide in the range between 52 em and ∞ em. This rule will never apply to values up to 52 em where the width could actually change thanks to the next query.